cluster-autoscaler/utils/test/test_utils.go

```diff
@@ -213,6 +213,12 @@ func TolerateGpuForPod(pod *apiv1.Pod) {
 
 // BuildTestNode creates a node with specified capacity.
 func BuildTestNode(name string, millicpu int64, mem int64) *apiv1.Node {
+	return BuildTestNodeWithAllocatable(name, millicpu, mem, millicpu, mem)
+}
+
+// BuildTestNodeWithAllocatable creates a node with specified capacity and allocatable.
+// When allocatable is negative number, the allocatable will be the same as capacity.
```

Comment:
I don't like this defaulting - test util functions should be as simple as possible, and this adds another layer. If I have to understand that negative values mean that capacity == allocatable, then I can just pass this value explicitely in test, making it more readable.